src/pages/index.tsx

```diff
@@ -40,38 +47,73 @@ export const ThemeChanger = () => {
 
 export const Footer = ({
   prev, next
-}: {prev: () => void, next:() => void}) => {
+}: { prev: () => void, next: () => void }) => {
   return (
     <div className="flex flex-row gap-3 my-6 justify-between px-6">
       <RetroButton type="button" onClick={() => prev()}>PREV</RetroButton>
       <div className="hidden md:block">
-      <ThemeChanger />
+        <ThemeChanger />
       </div>
       <RetroButton type="button" onClick={() => next()}>NEXT</RetroButton>
     </div>
   )
 }
 
-
-export default function Home() {
-  const {isFellow, fellow} = useIsFellow();
-  // non-fellow view
-  if (!isFellow || !fellow) {
-    return (
-      <Main>
-        <div className="p-5">
-          Hello! The Kernel Atlas is currently only accessible to the Kernel Fellows.
-        </div>
-      </Main>
-    )
+export async function getServerSideProps() {
+  return {
+    props: {
+      isuserFellow: (await isUserFellow('avirajkhare00@gmail.com')).isUserFellow,
```

Comment:
why is this hardcoded?

Reply:
I am not able to get email address of current logged in user. Since this piece of code is executed first. I won't be able to get email address here. Please help me here 🙏